ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/export/package.scala

```diff
@@ -9,6 +9,6 @@ package object export {
 
   type WriteItem = Raw.KeyValuePair
 
-  type WriteSet = Seq[WriteItem]
+  type WriteSet = collection.Seq[WriteItem]
```

Comment:
`Seq` defaults to `immutable.Seq` now but we put buffers in here.

Reply:
`mutable.Seq` means you can actually mutate it whereas `collection.Seq` just means that the underlying data structure can be mutable but you’re not going to mutate it. I find mutability scary so I’d rather keep `mutable.Seq` for cases where we actually mutate things.